sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/lang/Constants.kt

```diff
@@ -39,7 +39,7 @@ val VirtualFile.queriesName
 internal val SqlDelightFile.queriesName
   get() = "${virtualFile!!.nameWithoutExtension.decapitalize()}Queries"
 internal val SqlDelightFile.queriesType
-  get() = ClassName(packageName, "${virtualFile!!.nameWithoutExtension}Queries")
+  get() = ClassName(packageName, "${virtualFile!!.nameWithoutExtension.capitalize()}Queries")
```

Comment:
Should we also make an extension function with a locale just like it's done in Butterknife to avoid localization issues?

Reply:
not sure what you mean - can you link the function from butterknife you're referring to?

Reply:
https://github.com/JakeWharton/butterknife/pull/1420/files